AutoPark/CarRepair/Orders.cs

```diff
@@ -0,0 +1,67 @@
+﻿using AutoPark.CsvHelper;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace AutoPark.CarRepair
+{
+    public class Orders
+    {
+        private List<string> _details;
+
+        public Orders(string ordersPath)
+        {
+            _details = new List<string>();
+            OrdersCount = new Dictionary<string, int>();
+            LoadOrders(ordersPath);
+        }
+
+        public Dictionary<string, int> OrdersCount { get; private set; }
+
+        public void Print()
+        {
+            foreach(var key in OrdersCount.Keys)
```

Comment:
```suggestion
            foreach(var (key, value) in OrdersCount)
```
Use deconstructor. It`s better